lightning/config/config.go

```diff
@@ -229,11 +231,10 @@ func (cfg *Config) Load() error {
 		}
 	}
 
-	// If the level 1 compact configuration not found, default to true
+	// If the level 1 compact configuration not found, default to false
```

Comment:
If we're changing the default to `false` it doesn't need the pointer indirection anymore (but why changing it to `false` in this PR?)